sbin/common/common.sh

```diff
@@ -173,7 +173,25 @@ function setBootJdk() {
     fi
 
     echo "Guessing JDK_BOOT_DIR: ${BUILD_CONFIG[JDK_BOOT_DIR]}"
-    echo "If this is incorrect explicitly configure JDK_BOOT_DIR"
+    echo "If this is incorrect explicitly configure JDK_BOOT_DIR using --jdk-boot-dir"
+
+    # Calculate version number of boot jdk. Output on Windows contains \r, hence gsub("\r", "", $3).
+    export BOOT_JDK_VERSION_NUMBER=$("${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/java" -XshowSettings:properties -version 2>&1 | awk '/java.specification.version/{gsub(/1\./,"",$3);gsub("\r", "", $3);print $3}' | tr -d \\r)
```

Comment:
Why are you doing both `tr -d \\r` and `gsub("\r", "", $3)`. Both do the same.

Reply:
Bash is not my strongest language 😅. I thought tr and awk did different things. I'll remove it